io.catenax.user_estimated_loading/1.0.0/UserEstimatedLoading.ttl

```diff
@@ -0,0 +1,248 @@
+   bamm:characteristic :WeightFactor ;
+   bamm:exampleValue "60.0"^^xsd:decimal .
+
+:futureScenario a bamm:Property ;
+   bamm:preferredName "Future scenario"@en ;
+   bamm:description "The future scenario cotains one or more future usage profiles where the sum of percentage is 100"@en ;
+   bamm:characteristic :FutureScenarioList .
+
+:futureScenarioId a bamm:Property ;
+   bamm:preferredName "Future scenario ID"@en ;
+   bamm:description "Identification number of future scenarios"@en ;
+   bamm:characteristic :PositiveInteger ;
+   bamm:exampleValue "1"^^xsd:positiveInteger .
+
+:Mileage a bamm-c:Measurement ;
+   bamm:preferredName "Mileage @en"@en ;
+   bamm:description "Distance in kilometre"@en ;
+   bamm:dataType xsd:positiveInteger ;
+   bamm-c:unit unit:kilometre .
+
+:AgeInMonths a bamm-c:Duration ;
+   bamm:preferredName "Age in months"@en ;
+   bamm:description "Age of vehicle in month"@en ;
+   bamm:dataType xsd:nonNegativeInteger ;
+   bamm-c:unit unit:month .
+
+:KbaVariant a bamm-c:Enumeration ;
+   bamm:preferredName "KBA Variant @en"@en ;
+   bamm:description "Current version of the Enumeration is sub-set of list defined from the German Federal Office for motor vehicles."@en ;
+   bamm:dataType xsd:string ;
+   bamm:see <https://www.kba.de/DE/Statistik/Verzeichnisse/systematische_verzeichnisse_inhalt.html?nn=3505666> ;
+   bamm-c:values ( "Limousine" "Kombilimousine" "Pick-up" "Van" "SUV" "andere" ) .
+
+:NumberOfDriveAxle a bamm:Characteristic ;
+   bamm:preferredName "Number of drive axles"@en ;
+   bamm:description "Characteristic of number of drive axles"@en ;
+   bamm:dataType xsd:positiveInteger .
+
+:EnergySource a bamm-c:Enumeration ;
+   bamm:preferredName "Energy Source"@en ;
+   bamm:description "Enumeration of possible energy sources/fuel types of a vehicle. The list is defined from the German Federal Office for motor vehicles."@en ;
+   bamm:dataType xsd:string ;
+   bamm:see <https://www.kba.de/DE/Statistik/Verzeichnisse/systematische_verzeichnisse_inhalt.html?nn=3505666> ;
+   bamm-c:values ( "Unbekannt" "Diesel" "Benzin" "LPG" "Hybr.Benzin/E" "Hybr.Diesel/E" "0001" "0002" "0003" "0005" "0006" "0007" "0008" "0009" "0010" "0019" "0022" "0023" "0024" "0025" "0026" "0027" "0029" "0030" "0000" "Vielstoff" "Benzin/LPG" "Benzin/komp.Erdgas" "Erdgas NG" "Hybr.Vielstoff/E" "Hybr.Erdgas/E" "Benzin/Ethanol" "Hybr.LPG/E" "Hybr.B/E plug-in" "Hybr.D/E plug-in" "Hybr.LPG/E plug-in" "Hybr.V/E plug-in" "Hybr.NG/E plug-in" ) .
+
+:EnginePowerKW a bamm-c:Measurement ;
+   bamm:preferredName "Engine Power @en"@en ;
+   bamm:description "Engine power expressed in kilowatt"@en ;
+   bamm:dataType xsd:integer ;
+   bamm-c:unit unit:kilowatt .
+
+:Weight a bamm-c:Measurement ;
+   bamm:preferredName "Weight"@en ;
+   bamm:description "Weight of an object"@en ;
+   bamm:dataType xsd:double ;
+   bamm-c:unit unit:kilogram .
+
+:Percentage a bamm:Characteristic ;
+   bamm:dataType xsd:decimal .
+
+:Boolean a bamm:Characteristic ;

```

Comment:
if you have replaced the reference to this Characteristic in the properties above, you can remove this characteristic at all.